pydm/widgets/byte.py

```diff
@@ -0,0 +1,325 @@
+from ..PyQt.QtGui import QWidget, QTabWidget, QColor, QPen, QGridLayout, QLabel, QPalette, QFontMetrics, QPainter, QBrush, QStyleOption, QStyle
+from ..PyQt.QtCore import pyqtSignal, pyqtSlot, pyqtProperty, Qt, QStringList, QSize, QPoint
+from .channel import PyDMChannel
+import numpy as np
+
+class PyDMBitIndicator(QWidget):
+  def __init__(self, parent=None):
+    super(PyDMBitIndicator, self).__init__(parent)
+    self.setAutoFillBackground(True)
+    self.circle = False
+    self._painter = QPainter()
+    self._brush = QBrush(Qt.SolidPattern)
+    self._pen = QPen(Qt.SolidLine)
+    
+  def paintEvent(self, event):
```

Comment:
Would it be possible to take benefit of the PyDMDrawing classes in here or it is better to keep them decoupled?